tm2/pkg/amino/genproto/bindings.go

```diff
@@ -169,7 +169,7 @@ func generateMethodsForType(imports *ast.GenDecl, scope *ast.Scope, pkg *amino.P
 		))
 	}
 
-	//////////////////
+	// ////////////////
```

Comment:
eurgh

can we maybe make this `// -----------` so that even if fmt has to auto-fix it it's not an eyesore?

Reply:
I've tried to do a search and replace for these weird line comments:


4c4a878